dart/lib/src/enricher/_io_enricher.dart

```diff
@@ -0,0 +1,84 @@
+import 'dart:async';
+import 'dart:io';
+
+import '../protocol.dart';
+
+import 'enricher.dart';
+
+final Enricher instance = IoEnricher();
+
+/// Enriches [SentryEvents] with various kinds of information.
+/// Uses Darts [Platform](https://api.dart.dev/stable/dart-io/Platform-class.html)
+/// class to read information.
+class IoEnricher implements Enricher {
+  @override
+  FutureOr<SentryEvent> apply(SentryEvent event) {
+    final contexts = event.contexts.copyWith(
+      operatingSystem: _getOperatingSystem(event.contexts.operatingSystem),
+      runtimes: _getRuntimes(event.contexts.runtimes),
+    );
+
+    return event.copyWith(
+      contexts: contexts,
+      extra: _getExtras(event.extra),
+    );
+  }
+
+  List<SentryRuntime> _getRuntimes(List<SentryRuntime>? runtimes) {
+    final dartRuntime = SentryRuntime(name: 'Dart', version: Platform.version);
+    final flutterRuntime = SentryRuntime(name: 'Flutter');
+    if (runtimes == null) {
+      return [
+        dartRuntime,
+        flutterRuntime,
+      ];
+    }
+    runtimes.addAll([
+      dartRuntime,
+      flutterRuntime,
+    ]);
+    return runtimes;
+  }
+
+  Map<String, dynamic> _getExtras(Map<String, dynamic>? extras) {
+    final args = Platform.executableArguments;
+    final packageConfig = Platform.packageConfig;
+    final cpuCount = Platform.numberOfProcessors;
+    final environment = Platform.environment;
+    final locale = Platform.localeName;
+    final localHostname = Platform.localHostname;
+    // TODO What about:
+    // Platform.executeable
+    // Platform.resolvedExecuteable
+    // Platform.pathSeperator
+
+    final moreExtras = <String, dynamic>{
+      if (args.isNotEmpty) 'executableArguments': Platform.executableArguments,
+      if (packageConfig != null) 'packageConfig': packageConfig,
+      'numberOfProcessors': cpuCount,
+      if (environment.isNotEmpty) 'environment': environment,
+      'localName': locale,
```

Comment:
```suggestion
      'locale': locale,
```

Maybe something we can align here with .NET.
There we have an "unofficial context interface" [called "Current Culture"](https://docs.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.currentculture?view=net-5.0)

![image](https://user-images.githubusercontent.com/1633368/117208778-82d29980-adc3-11eb-81bb-79b346518b94.png)

Maybe we can get together to spec it out (aka: someone starts with a PR to https://github.com/getsentry/develop with a proposal) and we consider extensions to the protocol to all this data you were able to grab here.

Reply:
Done: https://github.com/getsentry/develop/pull/329